### DISCUSSION TITLE
Invalid type for `on`

### DISCUSSION THREAD

**Discussion by loctran016** [community/community#27063] [Actions]
<p>Hi. I’ve run into a problem:<br>
<a href="https://github.com/Cokaps/GenZOC/commit/98ec139ef559a0966e78f7c955cb7f68fdefbe19#annotation_1580187336" rel="noopener nofollow ugc"> <strong>.github/workflows/main.yml</strong> : .github#L1</a><br>
Invalid type for <code>on</code></p>
<p>Here is the <a href="https://github.com/Cokaps/GenZOC/blob/98ec139ef559a0966e78f7c955cb7f68fdefbe19/.github/workflows/main.yml" rel="noopener nofollow ugc">workflow</a>. Thanks in advance<br>
Edit: I’ve fixed this</p>

**Comment by Simran-B** [comment]
<p>Mind sharing what was wrong and how you fixed it with the community?</p>
<p>Also, your repository appears to be private (above links give me a 404 errors).</p>

**Comment by loctran016** [comment, marked as answer]
<p>It’s not private, I’ve changed my user name, deleted it and regenerate it with the same name (so Github don’t redirect, you can find it here: <a href="https://github.com/cokaps016/GenZOC" class="inline-onebox" rel="noopener nofollow ugc">GitHub - cokaps016/GenZOC: A simple tool to generate your OpenCore EFI automatically</a>.<br>
The fix is just syntax problem:<br>
This cause the problem (tags isn’t the child of <code>push</code>)</p>
<pre><code class="lang-auto">on:
  push:
    branches:
      - main
  tags:
      - v[0-9]+.[0-9]+.[0-9]+*
</code></pre>
<p>The way to fix this is easy, make <code>tags</code> into <code>push</code> child:</p>
<pre><code class="lang-auto">on:
  push:
    branches:
      - main
    tags:
      - v[0-9]+.[0-9]+.[0-9]+*
</code></pre>
